.github/workflows/release.yml

```diff
@@ -76,6 +76,7 @@ jobs:
           git config --global user.email "mkuznets@redhat.com"
 
           export GITHUB_TOKEN=${{ secrets.CHE_BOT_GITHUB_TOKEN }}
+          set -xe
```

Comment:
if flag is set in the Bash script we won't need it there

Reply:
true... again, not sure if we want to fail for everything that could go wrong in make-release (like all the error code 1 crap that `git` throws for benign warnings). So... yeah perhaps it's not needed here.

Reply:
but we should only call git commands if there is something to do, not expect it should fail and ignore error for some case